### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by IBUNHABIBU**
I have made the following changes
1. Adding the dropdown Select tag to select the status of the book whether is read or not
2. A read status button to change the status whether is  read or unread

### CODE REVIEW

**Comment by IBUNHABIBU** [comment]
I created the book library app with the features to add and remove books

File: library.js [hunk 1]
```diff
@@ -1,7 +1,158 @@
...
+// let clicked = false;
+// function toggle() {
+//   if (!clicked) {
+//     clicked = true;
+//     document.getElementById('toggle-btn').innerHTML = 'unread';
+//   } else {
+//     clicked = false;
+//     document.getElementById('toggle-btn').innerHTML = 'read';
+//   }
+// }
```
Comment (bellom) [inline library.js]: It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: booklibrary.html [hunk 2]
```diff
@@ -3,9 +3,59 @@
...
+            <div class="form-control">
+                <label for="author">Author</label>
+                <input type="author" id="author">
+                <small class="small-error">Error message</small>
+            </div>
+            <div class="form-control">
+                <label for="pages">No of Pages</label>
+                <input type="pages" id="pages">
+                <small class="small-error">Error message</small>
+            </div>
```
Comment (bellom) [inline booklibrary.html]: opps, users should not be able to input non-digit value.
image

Comment (bellom) [inline booklibrary.html]: Opps, we should have a checkbox in the form to specify the status of the book.

Users should be able to specify the status of the book either read or unread in the form.

image

https://www.theodinproject.com/courses/javascript/lessons/library

File: booklibrary.html [hunk 3]
```diff
@@ -3,9 +3,59 @@
...
+    integrity="sha512-xA6Hp6oezhjd6LiLZynuukm80f8BoZ3OpcEYaqKoCV3HKQDrYjDE1Gu8ocxgxoXmwmSzM4iqPvCsOkQNiu41GA=="
+     crossorigin="anonymous" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,300&display=swap" rel="stylesheet">
     <title>Document</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    
+    <div class="container mt-4">
+        <!-- <button id="toggle-btn" onclick="toggle();">Read</button> -->
```
Comment (bellom) [inline booklibrary.html]: Opps, the toggle feature is not working.
I'm unable to change the status of the book.

if you're not using this line of code, kindly delete it.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @IBUNHABIBU  :clap: 

**Quick One:** _**This project is one of the collaborative projects but you're working alone, may I ask why. kindly drop your answer in here for other TSE to see, thanks.**_

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by bellom** [comment]
> I created the book library app with the features to add and remove books

**Your Pull Request is not well-detailed and it shouldn't be a comment;**

image
Example; 
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

File: library.js (hunk 1 above)
Comment (IBUNHABIBU) [reply library.js]: I have no coding partner, I work alone in this project.

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Change Require✖️
## 🙋‍♂️ Kombo
#### You have a great on addressing the issues raised in the previous review, here is one thing you left out. 
* [  ] Kindly remove commented code from the ```booklibrary.html``` found on line 18.
``` <!-- <button id="toggle-btn" onclick="toggle();">Read</button>  --> ```
### That is it kindly address the issue and submit a re-review😎
#### Happy coding 👨‍💻
Addo Forison

**Review by shshamim63 — APPROVED** [review]
## :100: Approved :100: 
___
Hi @IBUNHABIBU :wave: ,
Awesome job :clap: The view is neat and clean, no commented code anymore. UI interacts properly. You have user local storage to store the list of books, great :mechanical_arm: You can work on it a little bit more to make it complete functional. Just add some book and delete any, then reload the page you will see it has a slight problem. But all the crucial changes are covered, so you have earned :trophy: .
___
#### This :+1: is from me. You can go ahead.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Pull request merged**